15-eco.Rmd

```diff
@@ -562,12 +562,12 @@ saveRDS(at, "extdata/15-tune.rds")
 ```
 
 ```{r 15-eco-26, echo=FALSE, eval=FALSE}
-tune = readRDS("extdata/15-tune.rds")
+at = readRDS("extdata/15-tune.rds")
```

Comment:
What does `at` stand for? Autotune? may be worth stating that somewhere or using a longer and more descriptive object name.

Reply:
yes, `at` stands for AutoTuner and I guess this was more obvious while creating the object: https://github.com/Robinlovelace/geocompr/blob/672991f23a7115b77c8ee199bdf8353b84f289f0/15-eco.Rmd#L526-L538
Still, the name could be more descriptive, something like `autotuner_rf` where rf stands for random forest.

Reply:
> Still, the name could be more descriptive, something like `autotuner_rf` where rf stands for random forest.

Agreed, I see now the tests are unrelated to your changes Jannes, so I suggest merging this now to keep the momentum. Many thanks!